src/libs/actions/Policy/Policy.ts

```diff
@@ -4610,6 +4612,242 @@ function deleteWorkspaceCompanyCardFeed(policyID: string, workspaceAccountID: nu
     API.write(WRITE_COMMANDS.DELETE_COMPANY_CARD_FEED, parameters, onyxData);
 }
 
+function unassignWorkspaceCompanyCard(workspaceAccountID: number, cardID: string, bankName: string) {
```

Comment:
We missed to update the optimistic data to apply online pattern B on this action